.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - main
+      - dev-*
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-binaries:
+    runs-on: ubuntu-latest
+    if: github.repository == 'numaproj-labs/numaplane'
+    name: Build binaries
```

Comment:
sorry for not having all of my findings at the same time, but I don't think "Build binaries" represents what's happening in this job

Reply:
I actually decided to remove the whole "Build binaries"  section as it seemed it was not doing anything substantial